pkg/server/grpc/tracee.go

```diff
@@ -42,10 +42,14 @@ func (s *TraceeService) StreamEvents(in *pb.StreamEventsRequest, grpcStream pb.T
 	for e := range stream.ReceiveEvents() {
 		// TODO: this conversion is temporary, we will use the new event structure
 		// on tracee internals, so the event received by the stream will already be a proto
-		eventProto := convertTraceeEventToProto(e)
+		eventProto, err := convertTraceeEventToProto(e)
+		if err != nil {
+			return err
+		}
```

Comment:
Failing here will break the receieve events loop - shouldn't we just log the error and continue?

Reply:
Yes, agreed.